Sources/Danger/GitHubDSL.swift

```diff
@@ -397,7 +397,7 @@ extension GitHub {
 
         public struct Label: Decodable, Equatable {
             /// The id number of this label.
-            public let id: Int // swiftlint:disable:this identifier_name
+            public let id: Int 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 400;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)